pylint/config/config_initialization.py

```diff
@@ -58,10 +58,18 @@ def _config_initialization(
     if reporter:
         linter.set_reporter(reporter)
 
+    # Set the current module to the command line
+    # to allow raising messages on it
+    linter.set_current_module("Command line")
+
     # Now we parse any options from the command line, so they can override
     # the configuration file
     parsed_args_list = linter._parse_command_line_configuration(args_list)
 
+    # Set the current module to configuration as we don't know where
+    # the --load-plugins key is coming from
+    linter.set_current_module("Configuration")
```

Comment:
```suggestion
    linter.set_current_module(f"Command line argument or {config_file}")
```

We can deduce the configuration file though.

Reply:
I think that might be a bit too long.

You would get 
```console
************* Module Command line argument or ~/Users/Daniel/Documents/MyHobbyProjects/MyProjectForNow/.pylintrc
Command line argument or ~/Users/Daniel/Documents/MyHobbyProjects/MyProjectForNow/.pylintrc:1:0: E0013: Plugin 'pylint_websockets' is impossible to load, is it installed ? ('No module named 'pylint_websockets'') (bad-plugin-value)
```

Do you have another, potentially shorter suggestion?

Reply:
I think it might have been relpath before ? (probably the reason why we have abspath and relpath in the expected result)
```suggestion
   linter.set_current_module(f"Command line argument or {config_file.relative_to(base_dir)}")
```

Reply:
Where are you getting `base_dir` from?

Reply:
That's the trick, nowhere 😄 I dont remember how I did it previously. Seeing https://github.com/PyCQA/pylint/blob/0fe866b8cc033334d4c438bcde92a324617ecd2b/pylint/testutils/configuration_test.py#L96 it was taken into account but it was probably handled in a pylint.config class on the code side.

Reply:
Yeah I know, but I don't really like that output now that I revisited this. Perhaps `Command line or configuration`?

Reply:
Look like it was a constant in the package originally : https://github.com/PyCQA/pylint/commit/c62738b66bcbe7220624b04192e9cfe6820ed13a#diff-5c78c16e096b746bf02709ba80aa1f9133f7bcd7b7360f042b14766ce7aaf881R15
I wonder how well it works when not developing locally, i.e. if pylint is installed in a venv or somewhere far from the directory the user is in.

Reply:
```suggestion
    linter.set_current_module("Command line or configuration file")
```